packages/ecommerce-app-base/src/Editor/SortableListItem.tsx

```diff
@@ -118,9 +119,9 @@ const CardDragHandle = SortableHandle(() => (
 ));
 
 export const SortableListItem = SortableElement<Props>(
-  ({ product, disabled, isSortable, onDelete }: Props) => {
+  ({ product, disabled, isSortable, onDelete, skuType }: Props) => {
     const [imageHasLoaded, setImageLoaded] = useState(false);
-    const [imageHasErrored, setImageHasErrored] = useState(false);
+    const [imageHasErrored, setImageHasErrored] = useState(!product.image);
```

Comment:
not sure I understand the reason for this change

Reply:
Hey John, without that code, when we pass a product that has no image (happens when for example the sku can no longer be found in the shop), the component unnecessarily renders and `img` with an empty sec, only to then immediately error and set this boolean to true. By immediately setting it to true, we prevent that unnecessary dance.

Just a small optimization, but not important if you think it makes the code harder to read.

Reply:
I might instead leave this as is, make the following change below - because it makes a bit more sense to me, and avoids `imageHasErrored` actually meaning two different things. I leave it up to you though :) 

```
          {!product.image || imageHasErrored ? (
            <div className={styles.errorImage}>
              <Icon icon={productIsMissing ? 'ErrorCircle' : 'Asset'} />
            </div>
          ) : (
            <div className={styles.imageWrapper(imageHasLoaded)}>
              <img
                style={{ display: imageHasLoaded ? 'block' : 'none' }}
                onLoad={() => setImageLoaded(true)}
                onError={() => setImageHasErrored(true)}
                src={product.image}
                alt={product.name}
                data-test-id="image"
              />
            </div>
          )}
```

Reply:
A great suggestion, implemented :)